vpr/CMakeLists.txt

```diff
@@ -25,11 +25,18 @@ file(GLOB_RECURSE LIB_SOURCES src/*/*.cpp)
 file(GLOB_RECURSE LIB_HEADERS src/*/*.h)
 files_to_dirs(LIB_HEADERS LIB_INCLUDE_DIRS)
 
+capnp_generate_cpp(CAPNP_SRCS CAPNP_HDRS
```

Comment:
Probably want some context comments about what this is doing. And how Capnproto works (external definitions, compiled to c++ linked to executable etc.)

Reply:
Done